astropy/table/column.py

```diff
@@ -89,6 +88,19 @@ def col_copy(col, copy_indices=True):
 
 
 class FalseArray(np.ndarray):
+    """
+    Class to create a stub ``mask`` property which is a boolean array of
```

Comment:
Done (I think).  Not sure what you mean about "periods at end of descriptions of parameters actually matter".  I did remove one period in a parameter description so they are all consistent (with no period).  I also noticed that sphinx takes the first full sentence as the method description in the summary table, so that was kind of working.  But yes, I got a little sloppy.